analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -971,3 +971,56 @@ struct StructWithValuePropertyAndCastOperators
     }
 }
 
+namespace TypeWithValueStaticProperty
+{
+    class Test
+    {
+        void Basics()
+        {
+            _ = AClassWithStaticValueProperty.Value;                                // Compliant, not on nullable value type
+            _ = AClassWithStaticValueProperty.Value.Value;                          // Compliant
+            _ = AClassWithStaticValueProperty.Value.Value.InstanceProperty;         // Compliant
+            _ = AClassWithStaticValueProperty.Value.Value.InstanceProperty.Value;   // Compliant
+            _ = new AClassWithInstanceValueProperty().Value;                        // Compliant
+        }
+    }
+
+    class AClassWithStaticValueProperty
+    {
+        public AClassWithInstanceValueProperty InstanceProperty => new AClassWithInstanceValueProperty();
+
+        public static AClassWithInstanceValueProperty Value => new AClassWithInstanceValueProperty();
+    }
+
+    class AClassWithInstanceValueProperty
+    {
+        public AClassWithStaticValueProperty Value => new AClassWithStaticValueProperty();
+    }
```

Comment:
These are not value properties.

Reply:
What is your definition of "value property"? It's a property, and it's named `Value`, so I named it ValueProperty.

Would you prefer `PropertyNamedValue` instead?

Reply:
Oh, this makes more sense. I was thinking of value types. I'd go with `ClassWithStaticPropertyCalledValue` to avoid confusion.